roles/convert/vars/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 # vars file for convert
+
+convert_convert2rhel_enable_repos_args: "{{ ('--enablerepo ' + convert_convert2rhel_repos_enabled | default([], true) | join(' --enablerepo ')) if convert_convert2rhel_repos_enabled | length > 0 else '' }}"
+convert_convert2rhel_disable_repos_args: "{{ ('--disablerepo ' + convert_convert2rhel_repos_disabled | default([], true) | join(' --disablerepo ')) if convert_convert2rhel_repos_disabled | length > 0 else '' }}"
```

Comment:
I don't know about this, i put it in there because both are possible in CLI. Looking now, the google doc task says 'Identify which repositories the user wants to enable', so maybe disablerepo is not really needed.

Reply:
I would still leave the disable repo option present here, as maybe, some users want to disable a couple of specific repositories. Might not be 100% needed for now, and enable repo might be more useful, but still, leaving disablerepo here is totally fine.